man/man5/mash_client.conf.5

```diff
@@ -0,0 +1,130 @@
+.\" Process this file with
+.\" groff -man -Tascii mash_client.conf.5
+.\"
+.TH mash_client.conf.5
+.SH NAME
+profile.yaml \- Configuration file for the mash client
+.SH DESCRIPTION
+The mash client is the command line tool to interact with a mash server.
+The mash command by default reads a configuration file from
+.IR ~/.config/mash.
+The mash client supports so called profiles. Each profile is represented by
+a configuration file in the configuration directory with the file name of
+.IR profile.yaml.
+This allows a user of mash to configure multiple target mash servers and/or
+multiple users on a given mash server, if the target mash server supports
+multiple user configuration. The support of multiple users on the mash server
+is controlled by the mash server administrator. In most cases the mash server
+will be configured to use OIDC (Open ID Connect) in which case only a single
+user account per mash user should exist on the mash server.
+
+The configuration file syntax is YAML style expressed as
+.IR key:value
+pairs.
+.PP
+email
+.RS 4
+Set the e-mail for the mash user account to be used for this profile for
+authentication to the mash server. The value should not be set if the mash
```

Comment:
It doesn't hurt for it to be set and the client will set the email after initial OIDC login.